<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroup</class>
 <widget class="QWidget" name="AddGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QComboBox" name="groupFather"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="groupNameL">
       <property name="text">
        <string>组织名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="groupFatherL">
       <property name="text">
        <string>父组织</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="groupName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="groupDesL">
       <property name="text">
        <string>描述</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="groupDes"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ok">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
